seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/enumerator/IncrementalSplitAssigner.java

```diff
@@ -278,4 +278,8 @@ public boolean completedSnapshotPhase(List<TableId> tableIds) {
         return context.getAssignedSnapshotSplit().isEmpty()
                 && context.getSplitCompletedOffsets().isEmpty();
     }
+
+    public boolean waitingForAssignedSplits() {
```

Comment:
Keep the logical in waitingForCompletedSplits?

Reply:
Stream log reading never ends, so not use waitingForCompletedSplits